HandheldCompanion/ViewModels/GameViewModel.cs

```diff
@@ -0,0 +1,297 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using craftersmine.SteamGridDBNet;
+using GameLib;
+using GameLib.Core;
+using GameLib.Plugin.RiotGames.Model;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using static Microsoft.WindowsAPICodePack.Shell.PropertySystem.SystemProperties.System;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace HandheldCompanion.ViewModels;
+
+public partial class GameViewModel : ViewModelBase
+{
+    const string CrossImagePath = "/Resources/GameLib/cross-color.png";
+    const string CheckImagePath = "/Resources/GameLib/check-color.png";
+
+    private readonly LauncherManager _launcherManager;
+
+    [ObservableProperty]
+    private ObservableCollection<GameExtended> _games = default!;
```

Comment:
Model object in collection, I don't like this..

Reply:
This was an interface. That i was busy extending to allow more properties. Also not happy with it and need to re think that

Reply:
I think you'll learn a lot from how I tackle MVVM.